### ISSUE TITLE
Simula builds from source

### ISSUE THREAD

**Issue by ltomes** [SimulaVR/Simula#171]
Hi there, I am hoping for a little guidance on getting a fast install. 
I'm on a fresh manjaro build with gnome.
My install steps match the readme:
```
git clone --depth 1 --recursive https://github.com/SimulaVR/Simula
cd Simula
source ./utils/Helpers.sh && installSimula
```
At which point simula doesn't find a cashed package, and starts the long process of building from source. Feel free to link me to any man pages or articles, I don't mind reading.

```
these 2 derivations will be built:
  /nix/store/2cnvlmbpk2dyvnk6jd78zv6c5mcl41iq-godot-3.2.drv
  /nix/store/hmcwa7ggi1qalf87j760q8bpdmykcmvq-Simula.drv
```

![image](https://user-images.githubusercontent.com/4184677/150620556-f78c5445-e280-458b-888a-ce32f8da4790.png)

Thanks for your time

**Comment by ltomes** [comment]
Anyone else who comes across this issue. If building Simula from source at least 16384m of ram _must_ be available or compile of godot will fail.

**Comment by georgewsinger** [comment]
@ltomes We seem to be having issues with caching lately. Did you manage to get it to build? If not, I just re-uploaded the binaries to cachix which should facilitate downloading/no build requirement.

**Comment by ltomes** [comment]
Yes I was able to build from source for QEMU/no gpu (or it's leveraging the 2080 hardware, and passing it through QEMU/novnc): 
![image](https://user-images.githubusercontent.com/4184677/153002735-c4351545-bff8-484f-a58a-b2e622504156.png)
